.github/workflows/operator-ci.yml

```diff
@@ -145,7 +145,6 @@ jobs:
               -Dquarkus.container-image.build=true \
               -Dquarkus.kubernetes.image-pull-policy=IfNotPresent \
               -Doperator.keycloak.image=keycloak:${{ env.version_remote }} \
-              -Dquarkus.jib.jvm-arguments="-Djava.util.logging.manager=org.jboss.logmanager.LogManager","-Doperator.keycloak.image-pull-policy=Never" \
```

Comment:
I believe the issues with the remote tests is because of this, we need to override the default behaviour for tests to actually never pull the KC image. Without it the tests just hang waiting for Keycloak pods to come up.

`operator.keycloak.image-pull-policy` is just a Quarkus config option (our custom one). You should be able to also override it by env vars that are set on the Operator pod.  I'll try to look into it.

Reply:
This might help: https://github.com/ASzc/keycloak/pull/2

Reply:
The tests seem to be passing on that PR, so I will merge it in. Thanks!